apps/prairielearn/python/zygote.py

```diff
@@ -84,6 +87,33 @@
 prairielearn.get_unit_registry()
 
 
+# We want to conditionally allow/block importing specific modules, namely `rpy2`.
+# This custom importer will allow us to do so, and throw a custom error message.
+#
+# While this won't prevent anything more complex than an `import` statement, it
+# will make it clear to the user that they're not allowed to use `rpy2`. If they
+# try to bypass the block, it's up to them to deal with the consequences.
+class ForbidModuleMetaPathFinder(MetaPathFinder):
+    def __init__(self):
```

Comment:
```suggestion
    def __init__(self) -> None:
```
Nit: All functions have the none return type specified

Reply:
Done in 04593ad.